.env.example

```diff
@@ -1,2 +1,6 @@
 CLIENT_ID=
-CLIENT_SECRET=
\ No newline at end of file
+CLIENT_SECRET=
+TENANT=
+FINDING_SEVERITY=
+FINDING_LIMIT=
+BASE_URL_API=
```

Comment:
This doesn't appear to be defined anywhere in the PR. What is the value supposed to be and how would a user know? Is this not a static value for all users?